pyodide_build/buildpkg.py

```diff
@@ -127,6 +127,7 @@ def package_files(buildpath, srcpath, pkg, args):
         'python',
         common.ROOTDIR / 'file_packager.py',
         name + '.data',
+        '--ABI=1997',
```

Comment:
We should get this from the top Makefile as well.  That probably means passing it in as a parameter here as well.

Reply:
Should I create a copy of this file in the ``src`` directory? Just like all the other files which use ``sed`` to introduce ABI as a parameter?

Reply:
No.  We should pass the ABI value into `buildpkg.py` as a commandline parameter.  (Add a parameter for it in the `make_parser` function, and then pass it in from all places that call `buildpkg`).